src/redux/mainWallet/selectors/index.js

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
Where did you get "mainWallet" word? :-) ChronoMint develop?

We have all "main" wallets in scope of 'ethereum' packages. Ethereum wallet is a parent of all other wallets.